index.js

```diff
@@ -0,0 +1,25 @@
+import './style.css';
+import TaskStatus from './modules/taskStatus.js';
+import tasks, { inputAdd, listContainer } from './modules/list.js';
+
+window.onload = tasks.displayList();
```

Comment:
You can use an Event Listener to render the list
```suggestion
document.addEventListener('DOMContentLoaded', () => {
  tasks.displayList();
});
```

Reply:
thank you so much for your review